db/migrate/20170721141401_create_taggings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTaggings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :taggings do |t|
+      t.integer :tag_id
+      t.integer :taggable_id
+      t.string :taggable_type
+
```

Comment:
@case-eee I decided to do tags as a polymorphic many-to-many with a joins table Taggings. I think I've got it set up right, but thought I'd tag you just in case.